compliance_checker/protocols/erddap.py

```diff
@@ -0,0 +1,42 @@
+import io
+import urllib.request
+import compliance_checker.protocols.opendap as opendap
+
+def is_tabledap(url):
+    """
+    Identify a dataset as an ERDDAP TableDAP dataset.
+
+    Parameters
+    ----------
+    url (str) : URL to dataset
+
+    Returns
+    -------
+    bool
+    """
+
+    if "tabledap" in url:
```

Comment:
I'd suggest using url.lower() here just to be safe.

Reply:
Just use `return "tabledap" in url.lower()` to simplify the return since it's returning boolean type.

Reply:
You also can't necessarily guarantee "tabledap" will be in the URL, e.g. Apache/Nginx setups.  Usually it will though.